VEHICLE/IsVehicleStoppedAtTrafficLights.md

```diff
@@ -9,10 +9,10 @@ BOOL IS_VEHICLE_STOPPED_AT_TRAFFIC_LIGHTS(Vehicle vehicle);
 ```
 
 ```
```

Comment:
Remove backticks, these are reserved for imported comments, not actual contributions by the community.